n2y/converter.py

```diff
@@ -0,0 +1,466 @@
+import importlib.util
+from os import path, makedirs
+from shutil import copyfileobj
+from urllib.parse import urlparse
+import requests
+
+from pandoc.types import Str, Para, Plain, Space, SoftBreak, Header, Strong, Emph, \
+    Strikeout, Code, CodeBlock, BulletList, OrderedList, Decimal, Period, Meta, Pandoc, Link, \
+    HorizontalRule, BlockQuote, Image, Underline, MetaString, Table, TableHead, TableBody, \
+    TableFoot, RowHeadColumns, Row, Cell, RowSpan, ColSpan, ColWidthDefault, AlignDefault, Caption
+import re
+
+from n2y.notion import Client
+
+
+# Notes:
+# A single Notion block may have multiple lines of text.
+# A page is a block that puts children into "content" attribute.
+# We transform page block to resemble other block types.
+#
+# Pandoc makes each word a block, and spaces are blocks too!
+#
+# Block types used here that do not exist in Notion:
+#   container - block with no top-level content, only children (used to parse a page and lists)
+#   bulleted_list - Notion has bulleted_list_item, but no enclosing container
+#   numbered_list - Notion has numbered_list_item, but no enclusing container
+
+
+IMAGE_PATH = None
+IMAGE_WEB_PATH = None
+
+
+def load_plugins(filename):
+    global Bookmark, BulletedList, BulletedListItem, ChildPageBlock, CodeBlockFenced, \
```

Comment:
I feel like there may be a way to do this without needing to repeat all of these names here and in the for loop... that said, this approach is straightforward and completely fine too---perhaps all of the alternatives are more complicated.

Reply:
That said, it wouldn't surprise me either if any solution we come up with has other downsides and the straightforward approach you have here is verbose but very simple. Thus, unless you're just curious now about finding another way, I'm fine just leaving as is too.

Reply:
I think using `globals()` with `issubclass()` would be a better solution than the current one. Using `issubclass( Block)` would prevent the plugins from overriding things they should not.